packages/frontend/src/components/PinnedItemsPanel/index.tsx

```diff
@@ -55,6 +60,40 @@ const PinnedItemsPanel: FC<Props> = ({ projectUuid, organizationUuid }) => {
                     : 'Your data team have pinned these items to help guide you towards the most relevant content!',
             }}
         />
+    ) : userCanUpdateProject && pinnedItems.length <= 0 ? (
+        // FIXME: update width with Mantine widths
+        <Card
+            w={900}
+            withBorder
+            style={{ backgroundColor: theme.colors.gray[1] }}
```

Comment:
it is better to use `sx` prop which has a theme awareness so you don't have to use `useMantineTheme`.

```tsx
sx={(theme) => ({
  backgroundcolor: theme.colors.gray[1],
})
```

Reply:
👍🏼